docs/libertyStart.md

```diff
@@ -5,16 +5,22 @@ Start a Liberty server in the background. The server instance is automatically c
 
 See the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration) properties for common server configuration.
 
-### Example  
-clean is set to `false` by default unless specified in `build.gradle` as shown in this example.  
+### Example
+`clean` is set to `false` by default unless specified in `build.gradle` as shown in this example.  
+To set up verification of applications installed from `installApps`, set `verifyAppStartTimeout` to how many seconds the server will wait in the message logs.
```

Comment:
set `verifyAppStartTimeout` to the number of seconds the server should spend checking for start messages in the logs before it times out.